.github/workflows/ci.yml

```diff
@@ -1534,6 +1538,9 @@ jobs:
               PLATFORM_ARGS+="-p qemu_x86 "
               PLATFORM_ARGS+="-p qemu_x86_64 "
               ;;
+            xtensa-amd_acp_6_0_adsp_zephyr-elf)
+              PLATFORM_ARGS+="-p acp_6_0 "
+              ;;
```

Comment:
e.g. Please comment this out if there is currently no upstream platform available for testing; otherwise, the test phase will fail.

```
              # xtensa-nxp_rt600_adsp_zephyr-elf is untested because no
              # upstream user platform is currently available.
              ;;
```